invenio_app_ils/eitems/loaders/jsonschemas/eitems.py

```diff
@@ -11,6 +11,19 @@
 from marshmallow import EXCLUDE, Schema, fields
 
 
+class IdentifierSchema(Schema):
```

Comment:
I think we have the same on the document can we reuse ? ping @ntarocco

Reply:
That's right, it's essentially `invenio_app_ils.documents.loaders.jsonschemas.document.IdentifierSchema`.